Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataTagsFactory.cs

```diff
@@ -12,25 +12,25 @@
 
 namespace EdFi.Ods.Features.OpenApiMetadata.Factories
 {
-    public class SwaggerTagsFactory
+    public class OpenApiMetadataTagsFactory
     {
-        private readonly ISwaggerFactoryResourceFilterStrategy _filterStrategy;
+        private readonly IOpenApiMetadataFactoryResourceFilterStrategy _filterStrategy;
 
-        public SwaggerTagsFactory(ISwaggerFactoryResourceFilterStrategy filterStrategy)
+        public OpenApiMetadataTagsFactory(IOpenApiMetadataFactoryResourceFilterStrategy filterStrategy)
         {
             _filterStrategy = filterStrategy;
         }
 
-        public IList<Tag> Create(IEnumerable<SwaggerResource> swaggerResources)
+        public IList<Tag> Create(IEnumerable<OpenApiMetadataResource> openApiMetadataResources)
         {
-            return swaggerResources.Where(x => _filterStrategy.ShouldInclude(x.Resource))
+            return openApiMetadataResources.Where(x => _filterStrategy.ShouldInclude(x.Resource))
                                    .Select(
                                         x => new Tag
-                                             {
-                                                 name = SwaggerDocumentHelper.GetResourcePluralName(x.Resource)
+                                        {
+                                            name = OpenApiMetadataDocumentHelper.GetResourcePluralName(x.Resource)
                                                                              .ToCamelCase(),
```

Comment:
this formatting needs to be fixed please

Reply:
Fixed